src/entitysdk/downloaders/emodel.py

```diff
@@ -0,0 +1,22 @@
+"""Download functions for EModel entities."""
+
+from pathlib import Path
+
+
+def download_hoc(client, emodel, hoc_dir="./hoc"):
```

Comment:
No need to be specific, output_dir would be sufficient here.

Reply:
done in latest commit